stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -543,6 +544,68 @@ public Token createPiiTokenSynchronous(@NonNull String personalId, String publis
                 mLoggingResponseListener);
     }
 
+    /**
+     * Blocking method to create a {@link Token} for a Connect Account. Do not call this on the UI
+     * thread or your app will crash. The method uses the currently set
+     * {@link #mDefaultPublishableKey}.
+     *
+     * @param accountParams params to use for this token.
+     * @return a {@link Token} that can be used for this account.
+     *
+     * @throws AuthenticationException failure to properly authenticate yourself (check your key)
+     * @throws InvalidRequestException your request has invalid parameters
+     * @throws APIConnectionException failure to connect to Stripe's API
+     * @throws APIException any other type of problem (for instance, a temporary issue with
+     * Stripe's servers)
+     */
+    public Token createAccountTokenSynchronous(@NonNull final AccountParams accountParams)
+            throws AuthenticationException,
+            InvalidRequestException,
+            APIConnectionException,
+            CardException,
+            APIException {
+        return createAccountTokenSynchronous(accountParams, mDefaultPublishableKey);
+    }
+
+    /**
+     * Blocking method to create a {@link Token} for a Connect Account. Do not call this on the UI
+     * thread.
+     *
+     * @param accountParams params to use for this token.
+     * @param publishableKey the publishable key to use with this request
+     * @return a {@link Token} that can be used for this account.
+     *
+     * @throws AuthenticationException failure to properly authenticate yourself (check your key)
+     * @throws InvalidRequestException your request has invalid parameters
+     * @throws APIConnectionException failure to connect to Stripe's API
+     * @throws APIException any other type of problem (for instance, a temporary issue with
+     * Stripe's servers)
+     */
+    public Token createAccountTokenSynchronous(
+            @NonNull final AccountParams accountParams,
+            @Nullable String publishableKey)
+            throws AuthenticationException,
+            InvalidRequestException,
+            APIConnectionException,
+            CardException,
+            APIException {
+        String apiKey = publishableKey == null ? mDefaultPublishableKey : publishableKey;
+        if (apiKey == null) {
+            return null;
+        }
+        validateKey(publishableKey);
+        RequestOptions requestOptions = RequestOptions.builder(
+                publishableKey,
+                null,
```

Comment:
The other create token methods pass in `mStripeAccount`. Why don't we want to do that here? (also, looks like there're constructors for `RequestOptions` that just take the key and request type, which seems possibly better than null)

Reply:
My thinking is that this is the function for creating a Stripe account so there should never be a pre-existing Stripe Account passed to it...since we're creating it with this function. In particular, I was concerned this would be cause unexpected server behavior since the token can be used for updates as well. Ex: somehow the user has an only StripeAccount string saved in the Stripe object and goes to create a new account with a new legal entity and accidentally changes their old legal entity. 
https://docs.google.com/document/d/1mmDgLjhbV8AuR_Ulxu6KULF4kemz3SNoUHTxco8b0u8/edit#heading=h.vzcoig4se8ry

I'll can go and double check if this is possible. I do agree it's inconsistent. 

on account updates:
https://docs.google.com/document/d/1mmDgLjhbV8AuR_Ulxu6KULF4kemz3SNoUHTxco8b0u8/edit

Reply:
Then I like this method better: https://github.com/stripe/stripe-android/blob/4163e57c1f431c85ca96190af0901ef15e4d0fe4/stripe/src/main/java/com/stripe/android/RequestOptions.java#L112-L114